config/core/300-resources/service.yaml

```diff
@@ -59,6 +59,1129 @@ spec:
+                            controller:
+                              type: boolean
+                              description: "If true, this reference points to the managing controller."
+                            kind:
+                              type: string
+                              description: "Kind of the referent. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds"
+                            name:
+                              type: string
+                              description: "Name of the referent. More info: http://kubernetes.io/docs/user-guide/identifiers#names"
+                            uid:
+                              type: string
+                              description: "UID of the referent. More info: http://kubernetes.io/docs/user-guide/identifiers#uids"
+                      resourceVersion:
+                        type: string
+                        description: "An opaque value that represents the internal version of
+                          this object that can be used by clients to determine when objects
+                          have changed. May be used for optimistic concurrency, change detection,
+                          and the watch operation on a resource or set of resources. Clients
+                          must treat these values as opaque and passed unmodified back to the
+                          server. They may only be valid for a particular resource or set of
+                          resources.
+                          \nPopulated by the system.
+                          \nRead-only.
+                          \nMore info: https://git.k8s.io/community/contributors/devel/api-conventions.md#concurrency-control-and-consistency"
+                      selfLink:
+                        type: string
+                        description: "SelfLink is a URL representing this object.
+                          \nPopulated by the system.
+                          \nRead-only.
+                          \nDEPRECATED"
+                      uid:
+                        type: string
+                        description: "UID is the unique in time and space value for this object.
+                          It is typically generated by the server on successful creation of
+                          a resource and is not allowed to change on PUT operations.
+                          \nPopulated by the system.
+                          \nRead-only.
+                          \nMore info: http://kubernetes.io/docs/user-guide/identifiers#uids"
+                  spec:
+                    type: object
+                    description: "Specification of the desired behavior of the pod."
+                    properties:
+                      containers:
+                        type: array
+                        minItems: 1
+                        maxItems: 1
```

Comment:
the multi-container feature flag when turned on will allow us to accept more than one container

Reply:
I didn't realize you could do that. I can remove that range.

Reply:
I removed `maxItems: 1` from here.